fbfmaproom/assets/style.css

```diff
@@ -50,8 +50,6 @@ table.supertable {
     border-spacing: 0;
     width: 100%;
     table-layout: fixed;
-    overflow-y: auto;
-    overflow-x: hidden;
```

Comment:
I don't think these ever had any effect, as they're overwritten by line 63.

Reply:
Yes. These were left over from some of my experiments, as far as I know they shouldn't do anything